public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please remove all the boilerplate code which is created by `Create React App`from `public` directory.

Reply:
thanks